src/Layout/redist/dnx.ps1

```diff
@@ -0,0 +1,6 @@
+# dnx.ps1
+# PowerShell script to launch dotnet.exe with 'dnx' and all passed arguments
+$scriptDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
```

Comment:
Users will need to select the ps1 script instead of the cmd, right? Is it ok that `DnxShimSource` can contain 2 possible shim sources now, and is there any logic that might need updating because of this?